eclair-core/src/main/scala/fr/acinq/eclair/router/RouteCalculationHandlers.scala

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2019 ACINQ SAS
```

Comment:
Much nitpick: you can use a dynamic year in Intellij.

Reply:
Doen with e646c4c .